### ISSUE TITLE
Roberta recipe in your paper is different from the original recipe

### ISSUE THREAD

**Issue by BaohaoLiao** [princeton-nlp/DinkyTrain#4]
Hi,

in Table 8 in https://arxiv.org/pdf/2202.08005.pdf, the recipe is different from the original Roberta recipe. Roberta_large uses a batch size of 8196, a peak learning rate of 4e-4 and trains for 100K steps. Your parameter setting seems from Table 3 in https://arxiv.org/pdf/1907.11692.pdf. But this recipe is used for Roberta_base instead of Roberta_large.

**Comment by CodeCreator** [comment]
Correct, we compare to the recipe in Table 3 of the RoBERTa paper, which can be understood as an improved BERT recipe with the same computational budget. Since BERT-large is trained with the same batch size and learning rate as BERT-base (https://arxiv.org/pdf/1810.04805.pdf), we also use the improved BERT recipe to train a large model.

**Comment by BaohaoLiao** [comment]
I see. For the Finetuning of your RoBERTa, do you use the hyperparameter setting in your paper or the one from the original paper?

**Comment by gaotianyu1350** [comment]
Hi,

We used pre-normed model for all our results in the paper. And we also use our own hyperparameter tuning grids as described in the paper.
